src/sql/base/browser/ui/taskbar/overflowActionbar.ts

```diff
@@ -188,12 +194,25 @@ export class OverflowActionBar extends ActionBar {
 	private moreElementOnClick(event: MouseEvent | StandardKeyboardEvent): void {
 		this._overflow.style.display = this._overflow.style.display === 'block' ? 'none' : 'block';
 		if (this._overflow.style.display === 'block') {
-			this._focusedItem = this._actionsList.childElementCount;
+			// set focus to the first element in the overflow
+			// separators aren't focusable so we don't want to include them when calculating the focused item index
+			this._focusedItem = this._actionsList.childElementCount - this.getActionListSeparatorCount();
 			this.updateFocus();
 		}
 		DOM.EventHelper.stop(event, true);
 	}
 
+	private getActionListSeparatorCount(): number {
+		let count = 0;
+		for (let i = 0; i < this._actionsList.children.length; i++) {
```

Comment:
This could be simplified 

`return this._actionsList.children.filter(c => c.className.includes('taskbarSeparator')).length;`